src/python/doc/barycenter_sum.inc

```diff
@@ -0,0 +1,24 @@
+.. table::
+   :widths: 30 50 20
+
+   +-----------------------------------------------------------------+----------------------------------------------------------------------+------------------------------------------------------------------+
+   | .. figure::                                                     | A Frechet mean (or barycenter) is a generalization of the arithmetic | :Author: Theo Lacombe                                            |
+   |      ./img/barycenter.png                                       | mean in a non linear space such as the one of persistence diagrams.  |                                                                  |
+   |      :figclass: align-center                                    | Given a set of persistence diagrams :math:`\mu_1 \dots \mu_n`, it is | :Introduced in: GUDHI 3.1.0                                      |
+   |                                                                 | defined as a minimizer of the variance functional, that is of        |                                                                  |
+   |      Illustration of Frechet mean between persistence           | :math:`\mu \mapsto \sum_{i=1}^n d_2(\mu,\mu_i)^2`.                   | :Copyright: MIT                                                  |
+   |      diagrams.                                                  | where :math:`d_2` denotes the Wasserstein-2 distance between persis- |                                                                  |
```

Comment:
Words split with '-' end up looking weird (same for "available" below), let's avoid that.

Reply:
Ok I will modify that.